Gemfile

```diff
@@ -30,11 +30,8 @@ gem "statsd-instrument", "~> 2.3.0"
 gem "unicorn", "~> 5.8.0"
 gem "validates_formatting_of"
 gem "elasticsearch", "~> 7.10.1"
-gem "elasticsearch-api", "~> 7.10.1"
```

Comment:
I have removed explicit api/transport requirement since we don't use it directly and it is already dependency of `elasticsearch` gem.